BackEndApp/BackEndApp/settings.py

```diff
@@ -27,6 +27,7 @@
 
 ALLOWED_HOSTS = [
     'localhost',
+    '127.0.0.1'
```

Comment:
Why was this needed?

Or was this needed?  or is it like you opened/tested on `127.0.0.1` instead of `locahost` so you just added it here?